frontend/src/app/main/ui/workspace/sidebar/options/rows/stroke_row.cljs

```diff
@@ -148,16 +148,17 @@
        [:& reorder-handler {:ref dref}])
 
      ;; Stroke Color
-     [:& color-row {:color (ctc/stroke->shape-color stroke)
-                    :index index
-                    :title title
-                    :on-change on-color-change-refactor
-                    :on-detach on-color-detach
-                    :on-remove on-remove
-                    :disable-drag disable-drag
-                    :on-focus on-focus
-                    :select-on-focus select-on-focus
-                    :on-blur on-blur}]
+     ;; FIXME: memorize stroke color
```

Comment:
Still a FIXME here

Reply:
Yes, It will be fixed eventually on other PR